Tests/ApolloCodegenTests/ApolloCodegenTests.swift

```diff
@@ -148,144 +148,77 @@ class ApolloCodegenTests: XCTestCase {
     )
 
     // then
-    expect(try ApolloCodegen.compileGraphQLResult(using: config).operations).to(beEmpty())
+    expect(try ApolloCodegen.compileGraphQLResult(config).operations).to(beEmpty())
   }
 
   // MARK: File Generator Tests
 
-  func test_fileGenerators_givenSchema_shouldCreateFileGeneratorsForUsedSchemaTypes() throws {
+  func test_fileGenerators_givenSchemaAndMultipleOperationDocuments_shouldGenerateSchemaAndOperationsFiles() throws {
```

Comment:
This test is about verifying the complete file list output and not about the file paths specifically, therefore I don't intend to write additional tests with different configurations. My next PR that address the correct subfolder structure for file output will test the varying combinations and ensure that the correct file paths are generated.